.dotnet/example_code/DynamoDB/TryDax/02-Write-Data.cs

```diff
@@ -23,8 +23,6 @@
  * specific language governing permissions and limitations under the License.
```

Comment:
Copyright notice goes here, see comment on first file.

Reply:
Revised as per discussion to match approved samples in TryDax.zip.